python/test/t/geometry/test_pointcloud.py

```diff
@@ -193,3 +193,26 @@ def test_pickle(device):
         assert pcd_load.point.positions.device == device and pcd_load.point.positions.dtype == o3c.float32
         np.testing.assert_equal(pcd.point.positions.cpu().numpy(),
                                 pcd_load.point.positions.cpu().numpy())
+
+
+def test_metrics():
+
+    from open3d.t.geometry import TriangleMesh, PointCloud, Metric, MetricParameters
+    # box is a cube with one vertex at the origin and a side length 1
+    pos = TriangleMesh.create_box().vertex.positions
+    pcd1 = PointCloud(pos.clone())
+    pcd2 = PointCloud(pos * 1.1)
+
+    # (1, 3, 3, 1) vertices are shifted by (0, 0.1, 0.1*sqrt(2), 0.1*sqrt(3))
+    # respectively
+    metric_params = MetricParameters(
+        fscore_radius=o3d.utility.FloatVector((0.01, 0.11, 0.15, 0.18)))
```

Comment:
Do we need o3d.utility.FloatVector or can we pass a python list/tuple?

Reply:
I think it's supposed to work automatically for STL containers and pybind11, but it doesn't work in Open3D for `std::vector<T>` due to some customizations. Going through a `deque` enabled it. :-(

Reply:
Is the problem with std::vector related to Eigen?

Reply:
Probably. We do some custom bindings for Eigen matrices / vectors as well as std::vector<int> and std::vector<double> That's likely disabling pybind11 default bindings. We should switch to pybind11's default bindings. [I don't know the original reason for customization, or if it's still needed.]